docker/automation/README.md

```
@@ -3,3 +3,8 @@
 Generic- and home-automation software.
 
 [How To Make Custom Tools in n8n to Automate ANYTHING with AI](https://www.youtube.com/watch?v=HsWwyipZNHw)
+
+## n8n
+
+Self-hosted AI starter kit
+https://github.com/bubacoder/coleam00-ai-agents-masterclass/blob/main/local-ai-packaged/Local_RAG_AI_Agent_n8n_Workflow.json
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)